### PR TITLE
Added help for MySQL State Store

### PR DESCRIPTION
**Pull request by DarqueWarrior**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

This is the help to accompany the new MySQL state store in [PR 612](https://github.com/dapr/components-contrib/pull/612)

## Issue reference

Please reference the issue this PR will close: #1100

### CODE REVIEW

**Review by orizohar — APPROVED** [review]
Great addition! Thank you!

**Comment by orizohar** [comment]
We should merge this after https://github.com/dapr/components-contrib/pull/612 is merged

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  metadata:
+  - name: connectionString
+    value: "<CONNECTION STRING>"
+  - name: actorStateStore
+    value: "true"
+```
+
+### Enforced SSL connection
+
+If your server requires SSL your connection string must end of `&tls=custom` for example, `"<user>:<password>@tcp(<server>:3306)/<database>?allowNativePasswords=true&tls=custom"`. You must replace the `<PEM PATH>` with a full path to the PEM file. If you are using [MySQL on Azure](http://bit.ly/AzureMySQLSSL) see the Azure [documentation on SSL database connections](http://bit.ly/MySQLSSL), for information on how to download the required certificate. The connection to MySQL will require a minimum TLS version of 1.2.
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md]:
```suggestion
If your server requires SSL your connection string must end with `&tls=custom` for example, `"<user>:<password>@tcp(<server>:3306)/<database>?allowNativePasswords=true&tls=custom"`. You must replace the `<PEM PATH>` with a full path to the PEM file. If you are using [MySQL on Azure](http://bit.ly/AzureMySQLSSL) see the Azure [documentation on SSL database connections](http://bit.ly/MySQLSSL), for information on how to download the required certificate. The connection to MySQL will require a minimum TLS version of 1.2.
```

File: daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "MySQL"
+linkTitle: "MySQL"
+description: Detailed information on the MySQL state store component
+---
+
+## Create a MySQL Store
+
+Dapr can use any MySQL instance. If you already have a running instance of MySQL, move on to the [Create a Dapr component](#create-a-dapr-component) section.
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md]: Should a tabbed view be here for both Self-Hosted and Kubernetes similar to other state store components?

Comment (DarqueWarrior) [reply daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md]: I copied from Postgre. I will look at others.

Comment (DarqueWarrior) [reply daprdocs/content/en/operations/components/setup-state-store/supported-state-stores/setup-mysql.md]: I will use Redis as my example going forward.

**Comment by DarqueWarrior** [comment]
The MySQL State store component was merged today.

**Comment by yaron2** [comment]
Congrats and thanks @DarqueWarrior

**Pull request merged**